package.json

```diff
@@ -32,6 +33,7 @@
   "homepage": "https://github.com/solid/chat-pane",
   "dependencies": {},
   "devDependencies": {
+    "@types/rdflib": "^1.0.5",
```

Comment:
Since you're re-exporting types from rdflib, I think it has to be a regular dependency, rather than a dev dependency - otherwise consuming projects will have it interpreted as `any`.

Reply:
I've pushed an update with that change -  it changed package-lock.json considerably, but from what I can see it looks reasonable

Reply:
OK, well... I guess it fits in line with Standard, then ¯\_(ツ)_/¯